extension/page-detect.js

```diff
@@ -58,6 +60,8 @@ window.pageDetect = (() => {
 		return isRepo() && blobPattern.test(location.href);
 	};
 
+	const hasCommentForm = () => isGist() || (isRepo() && (isIssue() || isNewIssue() || (isPR() && !isPRFiles()) || isCompare()));
```

Comment:
But it is. All of these pages (except Gist lists, but I could add an `isSingleGist` test) always have at least one comment form. On the compare page it's hidden by default, but it's still there. The only exception is if the user isn't logged in, which I think we can ignore given our target audience 😄